hibp.js

```diff
@@ -39,7 +39,7 @@ const HIBP = {
   async loadBreachesIntoApp(app) {
     console.log("Loading breaches from HIBP into app.locals");
     try {
-      const breachesResponse = await this.req("/breaches");
+      const breachesResponse = await this.req("/api/v2/breaches");
```

Comment:
This looks like you've got `HIBP_API_ROOT="https://haveibeenpwned.com/"` in your `.env`. It should be `HIBP_API_ROOT="https://haveibeenpwned.com/api/v2"` - i.e., the `api/v2` is in the environment variable.